packages/sdk/wallets/wallet-ledger/package.json

```diff
@@ -32,7 +32,7 @@
     "@celo/wallet-remote": "^6.0.2-beta.0",
     "@ethereumjs/util": "8.0.5",
     "@ledgerhq/errors": "^6.16.4",
-    "@ledgerhq/hw-app-eth": "git+https://github.com:celo-org/ledgerjs-hw-app-eth.git",
+    "@ledgerhq/hw-app-eth": "git+https://github.com:celo-org/ledgerjs-hw-app-eth.git#commit=b10b34511f12beb67488d3d6abab2e88192ae76a",
```

Comment:
we shoulda done this from the start. good call. 

can we also use this as a bundled dependency. https://docs.npmjs.com/cli/v10/configuring-npm/package-json#bundledependencies

reason being that git deps are considered less secure